CHANGELOG.next.asciidoc

```diff
@@ -386,6 +386,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add support for NATS version 2. {pull}13601[13601]
 - Add `docker.cpu.*.norm.pct` metrics for `cpu` metricset of Docker Metricbeat module. {pull}13695[13695]
 - Add `instance` label by default when using Prometheus collector. {pull}13737[13737]
+- Add small improvements in the azure module (validation, doc update) {pull}13859[13859]
```

Comment:
I have added both entries in the list, is there a reason the second one should be removed?

Reply:
@narph My understanding is changelog is user facing. Since this is not released yet, they don't need to know how many PRs we spent to add the initial pr, the improvements, the docs and etc. But I agree with Jaime, the new module needs to be an entry by itself 😬

Reply:
I have merged the 2 PRs in one comment line